templates/satspay/display.html

```diff
@@ -356,11 +256,22 @@
         if (!this.charge.payment_request && this.charge.hasOnchainStaleBalance)
           return
         try {
-          const {data} = await LNbits.api.request(
+          const { data } = await LNbits.api.request(
             'GET',
             `/satspay/api/v1/charge/balance/${this.charge.id}`
           )
           this.charge = mapCharge(data, this.charge)
+          if (this.charge.paid) {
+            this.timerInterval = setInterval(() => {
+              if (this.timer >= 100) {
+                clearInterval(this.timerInterval)
+                console.debug('redirecting')
```

Comment:
- why slow redirect?
- remove `console.log`

Reply:
give time for customer to know it's being redirected... else it's too fast, might be confusing!